config/plugins.js

```diff
@@ -13,7 +13,7 @@ module.exports = ({env}) => ({
     sentry: {
       enabled: true,
```

Comment:
Can we enable this based on environment? We only need to collect from staging and production.

Reply:
Oh never mind, development and testing uses the config in https://github.com/dev-launchers/strapiv4/tree/main/config/env.